#!/usr/bin/env python
PACKAGE = "magnification_prototypes"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("zoom_percentage", double_t, 0, "Percentage zoom on camera image.", 200, 100, 500)
gen.add("x_focus", double_t, 0, "X position of focal point, camera frame", 0, -10, 10)
gen.add("y_focus", double_t, 0, "Y position of focal point, camera frame", 0, -10, 10)
gen.add("z_focus", double_t, 0, "Z position of focal point, camera frame", 10, 0, 50)

exit(gen.generate(PACKAGE, "magnification_prototypes", "Zoom"))
